features/list-streams.feature

```diff
@@ -0,0 +1,10 @@
+Feature: List-streams
```

Comment:
```suggestion
Feature: List Streams

    As a User
    I want to see a list of my data streams
    So that i can analyze them

    Scenario: Successful List
    
        Given that I have signed-in
        And I am at the dashboard page
        And my streams have successfully loaded
        Then I expect to see a list of my data streams
```

Reply:
Also, check the reference I listed above, because I think the feature cant be described with the words you used, like `I`. There is some syntax highlighting on it

Reply:
Nevermind:
`These description lines are ignored by Cucumber at runtime, but are available for reporting (they are included by reporting tools like the official HTML formatter).`

Reply:
But I dont think it is a good description of a feature. Look a this one:
> Feature: Guess the word

> The word guess game is a turn-based game for two players.
> The Maker makes a word for the Breaker to guess. The game
> is over when the Breaker guesses the Maker's word.

So maybe we can make something like:

```suggestion
Feature: List Streams

    It enables the user to list all streams inside the Amazon Kinesis Streams.
    Further, the user can select one of these streams and manipulate or visualize it.

   Example: you can add one example here if you think it would be useful

   Scenario: Successful List
    
        Given that I have signed-in
        And I am at the dashboard page
        And my streams have successfully loaded
        Then I expect to see a list of my data streams
```